core/gather/driver/target-manager.js

```diff
@@ -40,16 +40,22 @@ class TargetManager extends ProtocolEventEmitter {
 
     this._enabled = false;
     this._rootCdpSession = cdpSession;
+    this._mainFrameId = '';
 
     /**
      * A map of target id to target/session information. Used to ensure unique
      * attached targets.
      * @type {Map<string, TargetWithSession>}
      */
     this._targetIdToTargets = new Map();
+    /** @type {Map<LH.Crdp.Runtime.ExecutionContextDescription['uniqueId'], LH.Crdp.Runtime.ExecutionContextDescription>} */
```

Comment:
nit: does this work?
```suggestion
    /** @type {Map<string, LH.Crdp.Runtime.ExecutionContextDescription>} */
```